hystrix/metric_collector/default_metric_collector.go

```diff
@@ -216,6 +216,10 @@ func (d *DefaultMetricCollector) UpdateRunDuration(runDuration time.Duration) {
 	d.runDuration.Add(runDuration)
 }
 
+// UpdateConcurrencyInUse updates concurrency in use.
+func (g *DefaultMetricCollector) UpdateConcurrencyInUse(concurrencyInUse float64) {
+}
```

Comment:
Can you add an implementation here like the other functions just for completeness.